js/forum/src/utils/DiscussionControls.js

```diff
@@ -55,14 +55,14 @@ export default {
         !app.session.user || discussion.canReply()
           ? Button.component({
             icon: 'reply',
-            children: app.trans(app.session.user ? 'core.forum.discussion_controls_reply_button' : 'core.forum.discussion_controls_log_in_to_reply_button'),
+            children: app.translator.translator.trans(app.session.user ? 'core.forum.discussion_controls.reply_button' : 'core.forum.discussion_controls.log_in_to_reply_button'),
```

Comment:
Whoopsie indeed! Thanks for catching that.

I started out by searching for "app.trans" instead of "app.trans(" and ended up applying the same change twice in a couple places. I thought I had got all of them. Sorry about that!